07_iifo/index.tex

```diff
@@ -1 +1,21 @@
-\chapter{Inter Island Forecasting Organisation (IIFO)}
\ No newline at end of file
+\chapter{Inter Island Forecasting Organisation (IIFO)}
+
+The role of IIFO is to allow islands to make predictions about the likelihood and severity of risks in the event of natural disasters, which correlates to the long term collective risk dilemma, and the returns from foraging, which correlates to the short term collective risk dilemma.
+
+\section{Long Term Collective Risk Dilemma (ltCRD)}
+\label{sec:IIFO:ltCRD}
+
+Islands may build a model to try and predict the likelihood and severity of upcoming disasters. They are then able to share this with the other islands along with a confidence level for their prediction, allowing the island to specify how much they believe their prediction to be the truth so that in the case that the island is wrong, agents might not judge them as harshly if their confidence level was low. On the flipside, a higher confidence level indicates that other islands should potentially place more trust in this prediction with the risk that the island offering the information will be judged more harshly if they turn out to be wrong. 
+
+The motivation behind sharing this information is that with more people contributing to the common pool, the more the easily the effects of the disaster can be mitigated. Sharing predictions allows the islands to better manage their resources as they have a better idea of when they should be donated to the common pool or not.
+
+Inaccurate predictions may mean that the islands are not prepared for a coming disaster, as it was predicted to come much later, meaning the common pool may not yet be large enough to mitigate the disaster effectively. Alternatively, the islands may over prepare and donate too much to the common pool when a disaster is not imminent, meaning they have unnecessarily reduced their own resources making it harder to forage and build up resources again.
+
+\section{Short Term Collective Risk Dilemma (stCRD)}
+\label{sec:IIFO:stCRD}
+
+Predicting the returns from foraging in different locations is beneficial to an island as it allows it to potentially maximise the returns if the prediction is accurate. IIFO facilitates the sharing of these predictions between islands.
+
+The benefit of sharing the prediction is due to the fact that typically foraging is performed by a group of islands. The more resources put in, the greater the returns for each resource. This means if an island is confident in where the best returns can be found, it is in its interest to let other islands know in the hope that they will also contribute to the foraging.
+
+If the prediction proves to be inaccurate, the returns will be less than expected and the islands that joined the foraging may lose faith in the original islands predictions making it harder for that island to convince the others to join it for foraging again in the future.
```

Comment:
suggested re-writing:

If the prediction proves to be inaccurate, the returns **from foraging** will be less than expected**,** and the islands that joined the foraging may lose faith in **the predictions of the island that originally proposed them**. **This may make it harder for the island whose predictions turn out to be wrong to convince others to forage together in the future.**